.circleci/config.yml

```diff
@@ -0,0 +1,184 @@
+            build_tag="${CIRCLE_SHA1}"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}"
+            docker build -t ${DOCKHUB_ORGANISATION}/ticktrade:${build_tag} -t ${DOCKHUB_ORGANISATION}/ticktrade:latest .
+      - run:
+          name: Pushing Image to docker hub
+          command: |
+            build_tag="${CIRCLE_SHA1}"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}"
+            echo $DOCKERHUB_PASSWORD | docker login -u $DOCKERHUB_USERNAME --password-stdin
+            docker push ${DOCKHUB_ORGANISATION}/ticktrade:${build_tag}
+            docker push ${DOCKHUB_ORGANISATION}/ticktrade:latest
+  k8s_deploy:
+    description: "Deploy to k8s cluster"
+    parameters:
+      target:
+        type: string
+    steps:
+      - k8s/install-kubectl
+      - run:
+          name: Deploying to k8s cluster for service binary-bot-beta
+          command: |
+            build_tag="${CIRCLE_SHA1}"
+            deployment_target="ticktrade-binary-com"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}" && deployment_target="ticktrade-binary-com"
+            echo $CA_CRT | base64 --decode > ca.crt
+            kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/${deployment_target} ${deployment_target}=${DOCKHUB_ORGANISATION}/ticktrade:${build_tag}
+
+  npm_test:
+    description: Run test
+    steps:
+      - run:
+          name: Run tests
+          command: |
+            for i in {1..3}; do npm test && break; done
+
+jobs:
+  test:
+    docker:
+      - image: circleci/node:11.15.0
+    steps:
+      - git_checkout_from_cache
+      - npm_install
+      - npm_test
+  release_beta:
+    docker:
+      - image: circleci/node:11.15.0
+    steps:
+      - git_checkout_from_cache
+      - npm_install
+      - build
+      - docker:
+          target: "beta"
+      - k8s_deploy:
+          target: "beta"
+  release_production:
+    docker:
+      - image: circleci/node:11.15.0
+    steps:
+      - git_checkout_from_cache
+      - npm_install
+      - build
+      - compress
+      - upload_artifact  # uploading the built code to s3 to create a backup of key services separate from Kubernetes deployment
+      - upload_checksum  # uploading compressed artifact checksum to cross match artifact fingerprint before actual deployment
+      - docker:
+          target: "production"
+      - k8s_deploy:
+          target: "production"
+
+workflows:
+  test:
+    jobs:
+      - test
+  release:
+    jobs:
+      - release_beta:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /V[0-9]+(\.[0-9]+)*-beta/
+      - release_production:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /V[0-9]+(\.[0-9]+)*/
+          context: ticktrade-binary-artifact-upload
```

Comment:
we have one context named `binary-frontend-artifact-upload` in production with required environment variables. if you suggest to create another with this name then you can create SRP in your card. or if you want to use that same context then you create one with same name in your circle account

Reply:
I changed it to `binary-frontend-artifact-upload`